src/app/pages/log-in/log-in.component.scss

```diff
@@ -0,0 +1,18 @@
+@import "typography";
+@import "color";
+
+.forgot-password {
+  display: flex;
+  justify-content: flex-end;
+  margin-top: 8px;
```

Comment:
margin-top is bad practice in most cases

Reply:
@dmitry-darly-solutions 
Identical to before question.

Reply:
@dmitry-darly-solutions 
Fixed.